server/controllers/user/login.php

```diff
@@ -49,6 +50,24 @@ public function validations() {
 
     public function handler() {
         $this->clearOldRememberTokens();
+
+        if ($this->checkGoogleLogin()) {
+            $client = new Google_Client(['client_id' => '50174278643-gtvjdpm5rmkv75lf3jsp95iv77a2usgu.apps.googleusercontent.com']);
```

Comment:
Why is this hardcoded? Do we have other step? this is not final correct?

Reply:
This is hardcoded but it isn't meant to be private. I agree we should place it somewhere else to make it customizable by people installing OS4, since every instalation will require different client ids.

Reply:
I plan to extend this by adding another input in the installation process and in the settings, would that be acceptable?